theSrc/scripts/LabeledScatter.js

```diff
@@ -176,20 +177,34 @@ class LabeledScatter {
           await FitLine.draw(this.rootElement, config)
         }
         if (config.quadrantsShow) {
+          const original_x_range = _.cloneDeep(plotlyChart._fullLayout.xaxis.range)
```

Comment:
Does it need a deep copy, isn't this just an array? i.e.
```suggestion
          const original_x_range = [...plotlyChart._fullLayout.xaxis.range]
```

Reply:
No, when I try it with a shallow copy the the array updates with the new values after I zoom. It needs the deep copy to keep the original values after the initial draw.